pkg/api/types.go

```diff
@@ -1824,6 +1824,9 @@ type PodSpec struct {
 	// If not specified, the pod will not have a domainname at all.
 	// +optional
 	Subdomain string `json:"subdomain,omitempty"`
+	// If specified, the pod's scheduling preferences
+	// +optional
```

Comment:
@smarterclayton @lavalamp What is the official decoration needed?  
xref: #34508